### DISCUSSION TITLE
What's the future of this repo?

### DISCUSSION THREAD

**Discussion by with-heart** [statelyai/xstate-tools#493] [General]
Since March of this year, there haven't been any PRs merged in this repo. There's been some conversation in issues but there also haven't been many issues opened since then.

I think it'd be helpful to decide on an official intention for this repo and the tools in it.

Having an official intention allows us to:

- add a status notice to the readme so users and contributors know what to expect
- respond to/update/close issues based on the intention
- decide whether to update or close existing PRs
- determine the work needed to move us into the future

Some relevant questions off the top of my head that need answered:

- do we plan to fix existing bugs in the v4 typegen?
- are we open to reviewing PRs from contributors that fix bugs or add features?
- should v5 users use the existing tooling?
- are there plans for new tooling for v5/v6?

**Comment by davidkpiano** [comment]
> do we plan to fix existing bugs in the v4 typegen?

There are currently no plans to fix existing bugs in v4 typegen.

> are we open to reviewing PRs from contributors that fix bugs or add features?

I would say so (cc. @Andarist)

> should v5 users use the existing tooling?

Also a good question for @Andarist 

> are there plans for new tooling for v5/v6?

This is something important to consider; there is a large amount of effort and commitment required for tooling for bidirectional code editing, even more than we previously thought; code can be written in many different ways and we need to either account for all of the use-cases/edge-cases, or document what is and isn't supported; both are tedious.

I think a more reasonable intermediate plan for the extension would be to support visualization of machines only, while we figure out a good path forward for bidirectional editing in XState v5 and beyond.
